samples/cellular/modem_shell/src/location/location_shell.c

```diff
@@ -660,6 +796,14 @@ static int cmd_location_get(const struct shell *shell, size_t argc, char **argv)
 		return -1;
 	}
 
+#if defined(CONFIG_LOCATION_DATA_DETAILS)
+	/* Store current command */
+	shell_command_str_from_argv(
+		argc, argv,
+		current_mosh_cmd,
+		LOCATION_DETAILS_CMD_STR_MAX_LEN + 1);
```

Comment:
The command generated here doesn't include `location` but subcommand `get` and all options.
This is seen in the cloud:
`"mosh_cmd":"get -m gnss --gnss_timeout 5 -m cellular -m wifi --cloud_details"`

Do we want to add command_prefix etc. parameter to `shell_command_str_from_argv()` that would be added into the beginning of the generated command?

Reply:
Added cmd_prefix parameter to `shell_command_str_from_argv`